.travis.yml

```diff
@@ -18,21 +18,24 @@ jobs:
       env: DEPENDENCIES='low'
     - php: 7.4
       env: SYMFONY_VERSION='^4.4'
+    - php: nightly
+      env: PHP8=true
 
 before_install:
   - phpenv config-rm xdebug.ini || echo "XDebug is not enabled"
 
 before_script:
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update "symfony/symfony:${SYMFONY_VERSION}"; fi;
   - if [ "$SKIP_INTEROP" != "" ]; then composer remove --dev --no-update container-interop/container-interop; fi;
-  - if [ "$DEPENDENCIES" != "low" ]; then composer update; fi;
+  - if [ "$DEPENDENCIES" != "low" ]; then composer update $([ -z "$PHP8" ] || echo "--ignore-platform-reqs") ; fi;
```

Comment:
I'd rather add support for a `$COMPOSER_FLAGS` variable (that could also solve the case of `--prefer-lowest`